documentation/tutorials/check.md

```diff
@@ -0,0 +1,174 @@
+
+## Configuration
+
+As well as the usual config parameters (`flyway.url`, `flyway.user`, `flyway.password`, `flyway.licenseKey`...), we also need to configure properties specific to the `check` command (see [the documentation](/documentation/command/check) for more details). In this case, we only need to configure `flyway.check.tempUrl` as the other properties all have suitable default values.
+
+Our `flyway.conf` file (found in the `conf` folder), should contain the following values:
+
+```
+flyway.url=jdbc:sqlserver://localhost;databaseName=foobar;trustServerCertificate=true
+flyway.user=sa
+flyway.password=Flyway123
+flyway.licenseKey=<put your license key here>
+
+flyway.check.tempUrl=jdbc:sqlserver://localhost;databaseName=check_temp_db;trustServerCertificate=true
+```
+
+## Set up some migrations
+
+In the `sql` folder, we can create a new file `V1__add_table.sql`, containing a simple schema change:
+
+```sql
+create table PERSON (
+    ID int not null,
+    NAME varchar(100) not null
+);
+```
+## Detecting changes using `-changes` flag
+
+Before we run `migrate`, we can use `check` to generate a change report by running it with the `-changes` flag:
+
+<pre class="console">./flyway check -changes</pre>
+
+## Viewing the report output
+
+Having run the command, we can see that Flyway has generated HTML and JSON versions of the `check` report in the working directory. Opening `check-report-<timestamp>.html` in a browser, we can clearly see what objects will change in our target schema as a result of running our pending migration(s), expressed in the form of SQL queries. The JSON report contains the same information but in a more machine-readable format. 
+
+<p align="center"><img src="/assets/tutorial/check/change-report-example.png" style="max-width: 100%"/></p>
+
+## Applying the changes
+
+We now know that applying our pending migration(s) will have the intended effect on our target schema, so we can go ahead and apply them using `migrate`:
+
+<pre class="console">./flyway migrate</pre>
+
+## Detecting drift using `-drift` flag
+
+Before applying future migrations, we might also want to confirm that no drift has occured in the target database, i.e. all schema changes have been applied in the form of flyway migrations, as oppose to unrecorded manual changes. 
+
+To demostrate this, we can apply a manual change to the target database, e.g. using SSMS:
+
+```sql
+use foobar;
+
+alter table PERSON add EMAIL varchar(100);
+```
+
+We can then run `check` with the `-drift` flag:
+
+<pre class="console">./flyway check -drift</pre>
+
+This creates another pair of HTML and JSON reports. This time the files contain a drift report, showing how the target database schema (in this case `foobar`) differs from whats contained in the applied migrations. 
+
+## Using Check in Docker
+
+This feature can also be accessed through the Flyway Enterprise Docker images.
```

Comment:
Could some sections be added to this part of the report? Like `### Pulling the images`, `### Running SQL Server in Docker` etc